docs/DART_LAB/matlab/oned_model_inf.m

```diff
@@ -1386,10 +1386,15 @@ function step_ahead(~, ~)
                     handles.inflation = str2double(get(handles.ui_edit_fixed_inflation,'String'));
                     
                 case 'Adaptive Inflation'
+                    % Here, only stick to one algorithm (Gaussian; falvor 2). Users may manually 
+                    % switch between 'Gaussian' [Anderson 2009] and 'Gamma'
+                    % [El Gharamti 2018]. The GUI option is only available
+                    % in the Lorenz'96 section. 
                     [lambda, handles.adap_inf_Std] = ...
                         update_inflate(mean(ens), var(ens), observation, obs_error_sd^2, inf_prior, ...
-                        handles.inflation, handles.adap_inf_Min, handles.adap_inf_Max, ...
-                        1, handles.adap_inf_Std, handles.adap_inf_Std_Min);
+                        handles.inflation, handles.adap_inf_Std, handles.adap_inf_Min, handles.adap_inf_Max, ...
+                        1, handles.adap_inf_Std_Min, handles.ens_size, 'Gaussian');
+                    
```

Comment:
Unfortunately, the matching Fortran routine is named 'update_inflation' not 'update_inflate' ... should we (being me) rename `private/update_inflate.m` to   `private/update_inflation.m` to match the Fortan? I think that's in the spirit of this pull request.

Reply:
Actually, this is not a one-to-one comparison. Forget it.